deps/wazuh_testing/wazuh_testing/qa_ctl/provisioning/qa_provisioning.py

```diff
@@ -105,6 +105,20 @@ def __process_config_data(self, host_provision_info):
             wazuh_branch = 'master' if 'wazuh_branch' not in deploy_info else deploy_info['wazuh_branch']
             s3_package_url = None if 's3_package_url' not in deploy_info \
                                         else deploy_info['s3_package_url']
+
+            version = None if 'version' not in deploy_info \
+                                else deploy_info['version']
+            repository = None if 'repository' not in deploy_info \
+                                else deploy_info['repository']
+
+            revision = None if 'revision' not in deploy_info \
+                                else deploy_info['revision']
```

Comment:
Consider that the schema is validating that these fields have value, so it may not be necessary to make this check when it is **required** in the schema.